frontend/public/i18n.js

```diff
@@ -0,0 +1,77 @@
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+import detector from 'i18next-browser-languagedetector';
+import httpBackend from 'i18next-http-backend';
+import Pseudo from 'i18next-pseudo';
+
+// Load moment.js locales (en is default and always loaded)
+import 'moment/locale/zh-cn';
+import 'moment/locale/ja';
+import 'moment/locale/en-gb';
+import moment from 'moment';
+
+import { FALLBACK_LOCALE } from './i18next-parser.config';
+const params = new URLSearchParams(window.location.search);
+const pseudolocalizationEnabled = params.get('pseudolocalization') === 'true';
+
+i18n
+  .use(new Pseudo({ enabled: pseudolocalizationEnabled, wrapped: true }))
+  // fetch json files
+  // learn more: https://github.com/i18next/i18next-http-backend
+  .use(httpBackend)
+  // detect user language
+  // learn more: https://github.com/i18next/i18next-browser-languageDetector
+  .use(detector)
+  // pass the i18n instance to react-i18next.
+  .use(initReactI18next)
+  // init i18next
+  // for all options read: https://www.i18next.com/overview/configuration-options
+  .init(
+    {
+      backend: {
+        loadPath: 'static/locales/{{lng}}/{{ns}}.json',
+      },
+      fallbackLng: FALLBACK_LOCALE,
+      load: 'all',
+      debug: process.env.NODE_ENV === 'development',
+      detection: { caches: [] },
+      ns: ['public', 'masthead', 'nav', 'activity-card', 'utilization-card', 'badge'],
+      defaultNS: 'public',
+      nsSeparator: '~',
+      keySeparator: false,
+      postProcess: ['pseudo'],
+      interpolation: {
+        format: function(value, format, lng, options) {
+          if (format === 'number') {
+            // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat#Browser_compatibility
+            return new Intl.NumberFormat(lng).format(value);
+          }
+          if (value instanceof Date) {
+            if (format === 'fromNow') {
+              return moment(value).fromNow(options.omitSuffix === true);
+            }
+            return moment(value).format(format);
+          }
+          return value;
+        },
+        escapeValue: false, // not needed for react as it escapes by default
+      },
+      react: {
+        useSuspense: true,
+        wait: true,
+      },
+      saveMissing: true,
+      missingKeyHandler: function(lng, ns, key) {
+        throw new Error(`Missing i18n key "${key}" in namespace "${ns}" and language "${lng}."`);
```

Comment:
Will this cause a whitespace or other user-visible error in console when it happens? If so, we should change how this works. We could just to set `window.windowError` here and make sure it's logged to the console.

Reply:
@spadgett - How do I look for a whitespace error? i18n is already configured to log the missing key to console.

Reply:
Sorry I meant whitescreen error. If I have a missing key, does it break console entirely if we throw an error here?

Reply:
It does. I can remove it.